examples/with-componentdidcatch/pages/index.js

```diff
@@ -1,4 +1,30 @@
-export default () => {
-  // Render time error
-  throw new Error('Test')
+import React, { Component } from 'react'
+
+class BuggyCounter extends Component {
```

Comment:
Awesome class name :joy:

Reply:
😂

(credit goes to [this demo](https://codepen.io/gaearon/pen/wqvxGa?editors=0010) from the react docs)